boom-app/src/router/index.ts

```diff
@@ -1,12 +1,12 @@
-import { createRouter, createWebHistory } from 'vue-router'
+import { createRouter, createWebHashHistory } from 'vue-router'
 import HomeView from '../views/HomeView.vue'
 import UploadView from '../views/UploadView.vue'
 import MappingView from '../views/MappingView.vue'
 import PreView from '@/views/PreView.vue'
 import { isUploaded } from '@/store'
 
 const router = createRouter({
-  history: createWebHistory(import.meta.env.BASE_URL),
+  history: createWebHashHistory(import.meta.env.BASE_URL),
```

Comment:
op zich is dit prima. als je WebHistory wil gebruiken dan moet je op de backend zorgen dat er een fallback is naar de index.html in Program.cs:
`app.MapFallbackToFile("/index.html");`